content/download/data.en.md

```diff
@@ -1,39 +1,89 @@
+</div>
+
+<hr>
+
+### <a name="NorthCarolinaDataset"></a> North Carolina data set
+
+This dataset is a modern package of geospatial data from North Carolina, USA. It offers raster, vector, LiDAR and satellite data.
+Most of the examples in the module manuals are written with this dataset. Furthermore, the tessuite operates with this dataset.
+The description, list of maps, and a quick usage tutorial are available here.
+The North Carolina dataset can be downloaded in two versions:
+<ul id="links" class="list-unstyled version">
+ <li>
+  <span class="mwl"><a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip " target="_blank"> nc_spm_full_v2alpha </a></span>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip" class="inl btn btn-primary" target="_blank">Download ZIP (159MB)</a>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz" class="inl btn btn-secondary">Download TAR.GZ (159MB)</a>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" target="_blank"> nc_basic_spm_grass7 </a></span>
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" class="inl btn btn-primary" target="_blank">Download ZIP  (50MB)</a>
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.tar.gz" class="inl btn btn-secondary">Download TAR.GZ  (50MB)</a>
+  </li>
+</ul>
+A smaller subset of the North Carolina location:
+<hr>
+
+
+### <a name="SpearFishDataset"></a> Spearfish dataset
+<hr>
+
+
+### <a name="SpecialTopicDatasets"></a> Datasets for special topics
+<hr>
+
+<ul id="links" class="list-unstyled version">
+ <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass79/manuals/index.html " target="_blank"> Data import </a></span>
+  <a href="https://grass.osgeo.org/grass79/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a> <a href="#" class="inl btn btn-secondary">Download ZIP</a>
+  <p>selected maps in common GIS formats (SHAPE, KML, GeoTIFF) including RGB orthophoto</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> LiDAR </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">LAS (21MB)</a> <a href="#" class="inl btn btn-secondary">LAZ (6MB)</a>
+  <p>extra Raleigh LiDAR dataset (note: height in feet):  multi-return</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Climate </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (690MB)</a>
+  <p>GRASS 7 Climatic data time series NC location nc_climate_spm_2000_2012 - related tutorial Separate NC climatic data location</p>

```

Comment:
link missing for the tutorial maybe?

Reply:
Yes, I have not yet managed to get to those details.